x-pack/plugins/ml/common/types/data_frame_analytics.ts

```diff
@@ -79,3 +79,9 @@ export interface DataFrameAnalyticsConfig {
   version: string;
   allow_lazy_start?: boolean;
 }
+
+export enum ANALYSIS_CONFIG_TYPE {
+  OUTLIER_DETECTION = 'outlier_detection',
+  REGRESSION = 'regression',
+  CLASSIFICATION = 'classification',
+}
```

Comment:
consider using const instead 
```suggestion
export const ANALYSIS_CONFIG_TYPE =  {
  OUTLIER_DETECTION: 'outlier_detection',
  REGRESSION: 'regression',
  CLASSIFICATION: 'classification',
} as const;
```

Reply:
Just talked to Melissa and since `ANALYSIS_CONFIG_TYPE` is used in a quite a lot of place, I think it's better to do this in a follow-up PR.